charts/oidc-proxy/templates/oidc-frontend-config.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.ingress.sslPolicy }}
+---
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: {{ template "oidc-proxy.fullname" . }}
+  labels:
+    {{- include "oidc-proxy.labels" . | nindent 4 }}
```

Comment:
Any chance this would need to be different labels than the other oidc component?